pyvista/core/pointset.py

```diff
@@ -325,7 +394,25 @@ def faces(self, faces):
             self.SetPolys(CellArray(faces))
 
     def is_all_triangles(self):
-        """Return ``True`` if all the faces of the ``PolyData`` are triangles."""
+        """Return ``True`` if all the faces of the ``PolyData`` are triangles.
+
+        Returns
+        -------
+        bool
+            ``True`` if all the faces of the ``PolyData`` are triangles.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> plane = pyvista.Plane()
+        >>> plane.is_all_triangles()
+        False
+
+        >>> sphere = pyvista.Sphere()
+        >>> sphere.is_all_triangles()
```

Comment:
```suggestion
        >>> sphere.is_all_triangles
```

Reply:
Right now, `is_all_triangles` is a method and not a property.  We should change this in a different PR because I think it makes sense to have it as a property.